### PR TITLE
feat:Avoid no content available page when URL permissions granted

### PR DESCRIPTION
**Pull request by jlperkins**
#### Description of changes

Changing behavior of the automated checks in FastPass when all URL permissions are granted to avoid showing the "no content available" page.  Addresses issue #2758 . Added variables to see if permissions were enabled and if the origin of the URL had changed. Amended unit tests as needed. 

#### Pull request checklist

- [x] Addresses an existing issue: #2758 
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [x] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Comment by jlperkins** [comment]
avoided
no content available
These are images of the details view as it is now when permissions are enabled and the target page is navigated away from, and then the "no content available" page that it previously showed before these changes.

**Comment by jlperkins** [comment]
no permissions, no content available
Gif of user running Automated Checks and then navigating away from the target page to another site.  The "no content available" page is displayed as a result, which is the expected behavior.

**Comment by jlperkins** [comment]
yes permissions, avoid no content available
Gif of user running Automated Checks and then clicks the "give Accessibility Insights additional permissions" link in the message about iframes.  User selects "allow" on the popup about permissions that appears.  Then, the user navigates away from the target page.  Instead of the "no content available" page, the Automated Checks page remains visible and a message about the target page having been changed is displayed, which is the expected behavior, since permissions were granted.

File: src/DetailsView/details-view-container.tsx [hunk 1]
```diff
@@ -85,13 +86,17 @@ export interface DetailsViewContainerState {
...
+    permissionsStateStoreData: PermissionsStateStoreData;
 }
 
 export class DetailsViewContainer extends React.Component<DetailsViewContainerProps> {
     private initialRender: boolean = true;
 
     public render(): JSX.Element {
-        if (this.isTargetPageClosed()) {
+        if (
+            !this.hasStores() ||
```
Comment (haonliu) [inline src/DetailsView/details-view-container.tsx:97]: NIT: would be good to extract this condition into a separate function as well.

**Pull request merged**